src/components/context/AuthContext.ts

```diff
@@ -25,19 +25,19 @@ const updateUser = (dispatch) => async (newUserData) => {
   }
 };
 
-const signin = (dispatch) => async (email, password) => {
+const signin = (dispatch) => async (email, password, staySignedIn) => {
   const data = await routeRequest('/api/v1/users/sign_in', { email, password }); // Sign-in first
-
-  if (data) {
-    const { id, display_name, email, token } = data;
+  const { id, display_name, userEmail, token } = data;
```

Comment:
Please revert these changes as the new structure of token data is handled in a further issue.